### DISCUSSION TITLE
Where is mDNS in 9.3.1

### DISCUSSION THREAD

**Discussion by MkLHX** [arendst/Tasmota#11230] [Q&A] [answered]
Hello, 
I set the SetOption55 to 1 by using command SO55 1 but it's seems start the mDNS in console but nothing in "informations" section and no mDNS resolution is possible on my devices.
How can i troubleshoot that?
Thx.

**Comment by sfromis** [comment, marked as answer]
You did not read the [changelog](https://github.com/arendst/Tasmota/blob/development/CHANGELOG.md#920-20201216)?
*mDNS has been disabled from all pre-compiled binaries to allow new features*

If you want to enable that problematic feature, you can include it when building your own binary.

**Reply by MkLHX** [reply]
Hi @sfromis thx far your answer. I don't read the chanlog because the SetOption55 is in the doc to enable mDNS. So I just hope that works as expected.
The mDNS was depreciate because it's difficult to maintain it on each platform?

**Reply by sfromis** [reply]
The maintenance of mDNS is not part of Tasmota, as it is implemented in Arduino Core. It proved to be buggy, and leading to problems for users. Saved quite a bit of code space to make it optional. I've never missed it.

As the docs covers all features of Tasmota, it is quite normal that some feature needs to be enabled for a documented command/option to apply.
